codejailservice/app.py

```diff
@@ -56,22 +54,41 @@ def configure_codejail(app):
                 limit_overrides_context=context,
             )
 
+
+# -----------------app configuration-------------------------
+
+app = Flask(__name__)
+LOG = logging.create_logger(app)
+env_config = os.getenv("FLASK_APP_SETTINGS",
+                       "codejailservice.config.ProductionConfig")
+app.config.from_object(env_config)
+
 configure_codejail(app)
 
-from codejail.safe_exec import SafeExecException, json_safe
-from codejail.safe_exec import not_safe_exec as codejail_not_safe_exec
-from codejail.safe_exec import safe_exec as codejail_safe_exec
+# Lazy imports of codejail functions according(after) the configuration  app.
+code_jail_safe_exec = importlib.import_module('codejail.safe_exec')
```

Comment:
This lazy import looks funky as well, maybe by moving to the other file we can make it cleaner

Reply:
For this, I make a method inside the `routes/code-exec.py` call `import_code_jail_safe_exec`.  e4c6724